Packs/Zscaler/ReleaseNotes/1_1_5.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Zscaler
+- Fixed an issue where **zscaler-category-add-url** and **zscaler-category-remove-url** were failing on large categories.
```

Comment:
```suggestion
- Fixed an issue where ***zscaler-category-add-url*** and ***zscaler-category-remove-url*** commands were failing on large categories.
```

Please make sure to follow the documentation https://xsoar.pan.dev/docs/documentation/release-notes#entities-marks-in-the-rn